docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -216,6 +220,87 @@ INFO[2023-04-24T23:19:59Z] Successfully installed YugabyteDB Anywhere!
 
 The `install` command runs all [preflight checks](#run-preflight-checks) first, and then proceeds to do a full install, and then waits for YBA to start. After the install succeeds, you can immediately start using YBA.
 
+## Migrate from Replicated
+
+{{< note >}}
+
+Migrating from Replicated using YBA Installer is [Early Access](/preview/releases/versioning/#feature-availability).
+
+{{</note >}}
+
+{{< warning title="Replicated end of life" >}}
+
+YugabyteDB Anywhere will end support for Replicated installation at the end of 2024.
+
+{{< /warning >}}
+
+If your YBA installation uses Replicated, you can use YBA Installer to migrate from Replicated.
+
+Review the [prerequisites](#prerequisites). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
+
+If you have high availability configured, you need to migrate your instances in a specific order. See [Migration and high availability](#migration-and-high-availability).
+
+### Migrate a YBA installation
+
+To migrate your installation from Replicated, do the following:
+
+1. If your Replicated installation is v2.18.5 or earlier, or v2.20.0, [upgrade your installation](../../../upgrade/upgrade-yp-replicated/) to v2.20.1.
+
+1. If you haven't already, [download and extract YBA Installer](#download-yba-installer).
+
+1. Optionally, configure the migration as follows:
+
+    ```sh
+    $ sudo ./yba-ctl replicated-migrate config
+    ```
+
+    This generates a configuration file `/opt/yba-ctl/yba-ctl.yml` with the settings for your current installation, which are used for the migration. You can edit the file to customize the install further. For a list of options, refer to [Configuration options](#configuration-options).
+
+1. Start the migration, passing in your license file:
+
+    ```sh
+    $ sudo ./yba-ctl replicated-migrate start -l /path/to/license
+    ```
+
+    The `start` command runs all [preflight checks](#run-preflight-checks) and then proceeds to do the migration, and then waits for YBA to start. After the migration finishes, YBA is installed.
+
+1. Validate YBA is up and running with the correct data, including Prometheus.
+
+    At this point, if you find problems, you can revert to your Replicated installation using the `replicated-migrate rollback` command. Note that any changes made with the new YBA (either using the UI or the API) will not be reflected after the rollback.
+
+    In particular, do not configure HA until running the `finish` command (next step) on all instances.
+
+1. If the new YBA installation is correct, finish the migration as follows:
+
+    ```sh
+    $ sudo ./yba-ctl replicated-migrate finish
+    ```
+
+    This uninstalls Replicated and makes the new YBA instance permanent.
+
+### Migration and high availability
+
```

Comment:
As a catchall let's highlight that failovers are NOT possible until after finalization  before we go into HTTP/HTTPS.